pkg/operations/operations.go

```diff
@@ -0,0 +1,104 @@
+package operations
+
+import (
+	"flag"
+	"fmt"
+	"sort"
+	"sync"
+)
+
+type Operation string
+
+const (
+	Audit   = Operation("audit")
+	Status  = Operation("status")
+	Webhook = Operation("webhook")
+)
+
+var (
+	// AllOperations is a list of all possible operations that can be assigned to
+	// a pod it is NOT intended to be mutated. It should be kept in alphabetical
+	// order so that it can be readily compared to the results from AssignedOperations
+	AllOperations = []Operation{
```

Comment:
I think it's valuable to export this so that users of operations can know the full suite of possible operations. We've used this pattern for things like constraint template status metrics in the past.

Reply:
un-exporting as we are not currently using it